src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,67 @@
+import { PeopleFilters } from '../../components/PeopleFilter/PeopleFilters';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api/api';
+import { Person } from '../../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [arePeopleLoading, setArePeopleLoading] = useState(false);
+  const [isLoadingSuccessful, setIsLoadingSuccessful] = useState(false);
+
+  const errorMessage = (
+    <p data-cy="peopleLoadingError" className="has-text-danger">
+      Something went wrong
+    </p>
+  );
+  const emptyPeopleMessage = (
+    <p data-cy="noPeopleMessage">There are no people on the server</p>
+  );
+
+  useEffect(() => {
+    setArePeopleLoading(true);
+    getPeople()
+      .then(peopleFromServer => {
+        setPeople(peopleFromServer);
+        setIsLoadingSuccessful(true);
+      })
+      .finally(() => {
+        setArePeopleLoading(false);
+      });
```

Comment:
u don't need it. if the request does not occur - the length will be === 0
```suggestion
  const [arePeopleLoading, setArePeopleLoading] = useState(false);

  const errorMessage = (
    <p data-cy="peopleLoadingError" className="has-text-danger">
      Something went wrong
    </p>
  );
  const emptyPeopleMessage = (
    <p data-cy="noPeopleMessage">There are no people on the server</p>
  );

  useEffect(() => {
    setArePeopleLoading(true);
    getPeople()
      .then(peopleFromServer => {
        setPeople(peopleFromServer);
      })
      .finally(() => {
        setArePeopleLoading(false);
      });
```